guide/13-managing-arcgis-applications/enterprise-sites-guide.ipynb

```diff
@@ -0,0 +1,578 @@
+{
```

Comment:
<p>ArcGIS Enterprise Sites allows you to create a tailored web page experience for your users by allowing you to share your portal's authoritative GIS data to other departments more easily, even if they are not used to working in your GIS. Instead of learning to navigate the portal and access groups, members can go directly to the custom web page you create to navigate the content relevant to them. Click <a href="https://enterprise.arcgis.com/en/sites/" target="_blank">here</a> to learn more about Enterprise Sites. </p><p>Note: The pattern to add, search, get, update, and delete <code>Sites</code> and <code>Pages</code> is identical to that of <strong>Initiatives</strong> (insert link to premium nb) as demonstrated here.  </p><p>In this notebook, we will demonstrate how to </p><ul><li>Create a site with a custom domain and pages</li><li>Edit a site's theme</li><li>Edit a site's page layout</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1450/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>